### ISSUE TITLE
Pause and then play any video it will skip some frames

### ISSUE THREAD

**Issue by MuhammadTalhaBaloch** [videojs/video.js#7127]
## Description
pause any video than play it again it will skip some frames 

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. visit https://videojs.com/advanced?video=disneys-oceans
2. pause the video than wait for some 1-2 seconds and play again
3. try 2nd step 2 or 3 times and it will surely reproduce

### Expected
frames shouldn't skip 

### Actual
frames are skipping

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
v7.11.4
#### browsers
Chromium(Microsoft Edge,Chrome)
#### OSes
windows 10
### plugins
are any videojs plugins being used on the page? If so, please list them below.

**Comment by gkatsev** [comment]
Are you able to record a video of what you're seeing? Given that we let the browser play this, and that you're seeing it in Edge as well as chrome, this is likely a browser bug. I'd expect the same issue to occur if you load the mp4 directly in a video element without Video.js.

**Comment by MuhammadTalhaBaloch** [comment]
I'm facing this issue on video js (edge and chrome browser) 
here are the recorded videos as you can see when i pause and than again play sometimes it will skip frames.

https://drive.google.com/drive/folders/1LzVxkIvIjOhwgL1MBUUkk-nYBsyqO1oi?usp=sharing

**Comment by gkatsev** [comment]
Thanks, I was just about to reproduce the skipped frames by loading the mp4 directly into Chrome, so, I think it's a Chrome issue and not a Video.js issue. Unfortunately, there isn't anything we can do about it.
It's worth opening up an issue against Chrome for this, if one doesn't already exist.

**Comment by MuhammadTalhaBaloch** [comment]
yes, i have opened ticket there lets hope that they are as active as video js community
